Library/Formula/couchdb.rb

```diff
@@ -98,6 +97,28 @@ def plist; <<-EOS.undent
   end
 
   def caveats; <<-EOS.undent
+    To start manually as the current user:
+        #{prefix}/bin/couchdb
+
+    If you want to automatically load `couchdb` on login:
+      First install:
+        mkdir -p ~/Library/LaunchAgents
+        cp #{prefix}/opt/couchdb/homebrew.mxcl.couchdb.plist ~/Library/LaunchAgents/
+        launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.couchdb.plist
+
+      Upgrade:
+        launchctl unload -w ~/Library/LaunchAgents/homebrew.mxcl.couchdb.plist
+        cp #{prefix}/opt/couchdb/homebrew.mxcl.couchdb.plist ~/Library/LaunchAgents/
+        launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.couchdb.plist
+
+    If you want to automatically load `couchdb` at boot:
+        # Note: Ensure that the couchdb user is set up according to 
+        # http://docs.couchdb.org/en/latest/install/mac.html#installation-with-homebrew
+        sudo launchctl list org.apache.couchdb \>/dev/null 2\>\&1 \&\& \\
+          sudo launchctl unload -w /Library/LaunchDaemons/org.apache.couchdb.plist
+        sudo cp #{prefix}/Library/LaunchDaemons/org.apache.couchdb.plist /Library/LaunchDaemons/
+        sudo launchctl load -w /Library/LaunchDaemons/org.apache.couchdb.plist
```

Comment:
I'm not sure I see a need to have this as well as the autogenerated caveats. Loading it as boot doesn't seem to be a Homebrew concern.